### PR TITLE
Refactor endpoints

### PR DESCRIPTION
**Pull request by brettmortensen**
Moves `json_decode` to `makeRequest` to avoid having each caller call
it. Cleans up string formatting to use double quotes instead of string
concatenation.

@sambev

### CODE REVIEW

File: atrium/AtriumClient.php [hunk 1]
```diff
@@ -47,13 +47,13 @@ function updateUser($userGUID, $params) {
...
     $json = json_encode($user);
 
-    $response = $this->makeRequest('PUT', '/users/' . $userGUID, $json);
-    return new User(json_decode($response, true)['user']);
+    $response = $this->makeRequest('PUT', "/users/$userGUID", $json);
+    return new User($response['user']);
   }
 
   function listUsers() {
     $response = $this->makeRequest('GET', '/users', '');
```
Comment (sambev) [inline atrium/AtriumClient.php]: What is the third `''` argument to `makeRequest` for?

Comment (sambev) [reply atrium/AtriumClient.php]: Looking at the code, seems like this is for the body. You can assign it a default argument of `''` so you don't have to pass it every time.

```php
private function makeRequest($method, $endpoint, $body = "") {
```

Comment (brettmortensen) [reply atrium/AtriumClient.php]: Great catch. For some reason I originally thought default args were not supported in PHP. Updated 👍

**Comment by sambev** [comment]
:shipit: Nice work!

**Pull request merged**